config/environments/development.rb

```diff
@@ -3,6 +3,8 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  config.action_mailer.default_url_options = { host: 'localhost', port: 5432 }
+
```

Comment:
[optional]: Kindly I would suggest you change the port to 3000, so that it can be the same as the rails server port.

Reply:
Thank you for the review. I have made the change at this [commit](https://github.com/Lucash2022/Blog_App/pull/12/commits/397449117b4f8a3526dc57c3be7f0d7218dd2a3b)